cmd/radio.go

```diff
@@ -185,18 +191,55 @@ type erasureConfig struct {
 	clnts  []bucketClient
 }
 
+func clientID(cfg remoteConfig) string {
+	hash := sha256.New()
+	jsonBytes, err := json.Marshal(cfg)
```

Comment:
rather than json perhaps use msgpack as well? we just need a binary representation